griptape/utils/constants.py

```diff
@@ -0,0 +1,2 @@
+class Constants:
+    RESPONSE_STOP_SEQUENCE = "<|Response|>:"
```

Comment:
Can you please add a class docstring that explains where this specific token comes from?